python/tvm/topi/scatter_elements.py

```diff
@@ -127,9 +127,9 @@ def gen_ir(data_ptr, indices_ptr, updates_ptr, out_ptr):
                 elif reduction == "mul":
                     out[index2] *= updates[index1]
                 elif reduction == "min":
-                    tir.min(out[index2], updates[index1])
+                    out[index2] = tir.min(out[index2], updates[index1])
                 elif reduction == "max":
-                    tir.max(out[index2], updates[index1])
+                    out[index2] = tir.max(out[index2], updates[index1])
```

Comment:
Also [PR#14008](https://github.com/apache/tvm/pull/14008)?

Reply:
Yes, but the problem was observed in this branch. After fix I repeated it on #14008 side. I think the latter will be merged first and I update this PR with mean reduction